packages/core/src/dialects/db2/connection-manager.ts

```diff
@@ -54,7 +54,7 @@ export class Db2ConnectionManager extends AbstractConnectionManager<Db2Connectio
       UID: config.username,
       // @ts-expect-error -- Bad typings
       PWD: config.password,
-      ...(config.ssl ? { SECURITY: config.ssl } : undefined),
+      ...(config.ssl ? { Security: 'SSL' } : undefined),
```

Comment:
Why not SECURITY?

Reply:
`SECURITY: 'SSL'` also works, though `Security` is more in line with the ['ibm_db' API Documentation](https://github.com/ibmdb/node-ibm_db/blob/master/APIDocumentation.md):
```bash
connStr = "DATABASE=database;HOSTNAME=hostname;PORT=port;Security=SSL;SSLServerCertificate=<cert.arm_file_path>;PROTOCOL=TCPIP;UID=username;PWD=passwd;";
```
Also: if someone finds the above and adds both `ssl: true` (_Sequelize docs_) to `options` and `Security: 'SSL'` (_ibm_db API docs_) to `dialectOptions` (_like I did initially_) then there will be **two SeCuRiTy props** inside the `connectionConfig` Object (_given_ `{config.ssl ? { SECURITY: 'SSL' }` _inside connection-manager_):
 
![image](https://github.com/sequelize/sequelize/assets/3778969/14cdedca-7bd5-4521-b97c-7ffe9831c6e4)
This will not work.

So I think it's better to have just `ssl: true` as a more standard syntax (`Security` _keyword gets set under the hood to the correct_ `'SSL'` _value_) + if someone happens to add `Security: 'SSL'` to `dialectOptions` assuming it was something dialect specific that needs to be specified, then nothing bad will happen.

This is just a quick fix to be able to connect to a database in the first place, I believe the whole `connectionConfig` block could be improved, see e.g. the TODO comment.